pxd_fastpath.c

```diff
@@ -515,6 +515,49 @@ void pxd_fastpath_adjust_limits(struct pxd_device *pxd_dev, struct request_queue
 	disableFastPath(pxd_dev, false);
 }
 
+// reset device called during device cleanup actions from any internal state.
+// consider node wipe, device remove while suspended etc.
+void pxd_fastpath_reset_device(struct pxd_device *pxd_dev)
```

Comment:
Doesn't this also need to be called from pxd_abort_context?

Reply:
yes it could help, revised.